docs/InstallAkita.md

```diff
@@ -0,0 +1,22 @@
+# Installing Akita
+
+Akita has been tested in Mozilla Firefox and Google Chrome. Instructions to install on these browsers is provided below. We hope to release Akita to browser extension stores in the near future.
+
+## Step 1: Download Akita Source
+From your preferred directory, run the command:
+
+`git clone https://github.com/dog-s/akita.git`
+
+Note the path to the `akita` directory.
+
+## Step 2: Install Akita in Your Browser
+
+### Installing Akita in Google Chrome
+1. In your Chrome address bar, type in `chrome://extensions/`. Press enter.
```

Comment:
Before this step, the user must make sure the "Developer mode" toggle in the top-right of `chrome://extensions/` is enabled.